crates/voicevox_core_c_api/src/drop_check.rs

```diff
@@ -0,0 +1,126 @@
+use std::{
+    collections::BTreeSet,
+    ffi::{c_char, CStr, CString},
+    sync::Mutex,
+};
+
+/// `*mut c_char`を`CString`としてdropしていいかチェックする。
```

Comment:
この書き方だと、CStringとして正しい`char*`かのチェッカーと勘違いされそうに思いました！
何をチェックするのか明示すると、こうとかでしょうか。
```suggestion
/// dropして良い`*mut c_char`を把握し、チェックする。
```